### ISSUE TITLE
openerp.modules.module: name 'hw_proxy' is not defined

### ISSUE THREAD

**Issue by tobru** [itpp-labs/pos-addons#710]
Trying to get `hw_printer_network` up and running leads into this error message when starting up Odoo / PosBox:

```
posbox_1  | 2018-07-05 19:26:43,862 1 INFO ? openerp: OpenERP version 8.0-20171009
posbox_1  | 2018-07-05 19:26:43,862 1 INFO ? openerp: addons paths: ['/var/lib/odoo/addons/8.0', u'/mnt/extra-addons', u'/usr/lib/python2.7/dist-packages/openerp/addons']
posbox_1  | 2018-07-05 19:26:43,862 1 INFO ? openerp: database hostname: db
posbox_1  | 2018-07-05 19:26:43,862 1 INFO ? openerp: database port: 5432
posbox_1  | 2018-07-05 19:26:43,862 1 INFO ? openerp: database user: odoo
posbox_1  | 2018-07-05 19:26:44,124 1 ERROR ? openerp.addons.hw_scanner.controllers.main: Odoo module hw_scanner depends on the evdev python module
posbox_1  | 2018-07-05 19:26:44,136 1 ERROR ? openerp.addons.hw_escpos.controllers.main: ESC/POS cannot initialize, please verify system dependencies.
posbox_1  | 2018-07-05 19:26:44,140 1 CRITICAL ? openerp.modules.module: Couldn't load module hw_printer_network
posbox_1  | 2018-07-05 19:26:44,140 1 CRITICAL ? openerp.modules.module: name 'hw_proxy' is not defined
posbox_1  | 2018-07-05 19:26:44,140 1 ERROR ? openerp.service.server: Failed to load server-wide module `hw_printer_network`.
posbox_1  | Traceback (most recent call last):
posbox_1  |   File "/usr/lib/python2.7/dist-packages/openerp/service/server.py", line 882, in load_server_wide_modules
posbox_1  |     openerp.modules.module.load_openerp_module(m)
posbox_1  |   File "/usr/lib/python2.7/dist-packages/openerp/modules/module.py", line 322, in load_openerp_module
posbox_1  |     getattr(sys.modules['openerp.addons.' + module_name], info['post_load'])()
posbox_1  |   File "/mnt/extra-addons/hw_printer_network/__init__.py", line 5, in post_load
posbox_1  |     from . import controllers
posbox_1  |   File "/mnt/extra-addons/hw_printer_network/controllers/__init__.py", line 4, in <module>
posbox_1  |     from . import hw_printer_network_controller
posbox_1  |   File "/mnt/extra-addons/hw_printer_network/controllers/hw_printer_network_controller.py", line 162, in <module>
posbox_1  |     hw_proxy.drivers['escpos_network'] = network_driver
posbox_1  | NameError: name 'hw_proxy' is not defined
posbox_1  | 2018-07-05 19:26:44,141 1 INFO ? openerp.service.server: HTTP service (werkzeug) running on 0.0.0.0:8069
```

Using the following command parameter: `--load=web,hw_proxy,hw_posbox_homepage,hw_scanner,hw_escpos,hw_printer_network` and the following Git commit: ac8f469f39378f910b995a5d028771a5a6b5ab15.

Any idea why `hw_printer_network` can't be loaded?

**Comment by GabbasovDinar** [comment]
@tobru what version of the module are you using? Please note the hw_printer_network module available on pos-addons since 10.0 version. Also probably you have old version of PosBox image. You can download last version of posbox_image https://nightly.odoo.com/master/posbox/

**Comment by tobru** [comment]
I'm running it in Docker and some dependencies where missing. Now it's running.
